tests/meson.build

```diff
@@ -57,7 +57,7 @@ endif
 
 test_uint128 = executable(
     'test-uint128',
-    ['test-uint128.c', '../util/types.c'],
+    ['test-uint128.c', '../util/types.c', '../util/suffix.c'],
```

Comment:
This looks like a leftover. As the unit test is added in patch #3.

Reply:
No 'int128_to_si_string' added in patch #1 uses `suffix_si_get_ld` so needed this to add and a build error occurred if not added this.

Reply:
Ah got it. :)